website-next/pages/preview/[id].tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react'
+import { useRouter } from 'next/router'
+import { PreviewPage } from '../../components/preview/preview-page-component'
+
+export default function Preview() {
+  const router = useRouter()
+  const { id } = router.query
```

Comment:
routing in Next.js is expressed via the folder/file structure. `pages/preview/[id].tsx` is the same as `/preview/[id]` was in React Router. To get the id parameter, I use `useRouter()` from next. The same `router.query` objects would also contain the query parameters passed to the site.